src/pybind/mgr/dashboard/services/rgw_client.py

```diff
@@ -2071,31 +2071,65 @@ def remove_sync_policy_group(self, group_id: str, bucket_name=''):
+            try:
+                exit_code, _, err = mgr.send_rgwadmin_command(rgw_sync_policy_cmd)
+                if exit_code > 0:
+                    raise DashboardException(f'Unable to create sync flow: {err}',
+                                             http_status_code=500, component='rgw')
+            except SubprocessError as error:
+                raise DashboardException(error, http_status_code=500, component='rgw')
+
         else:
-            if zones:
-                rgw_sync_policy_cmd += ['--zones', ','.join(zones)]
+            if zones is not None and (zones['added'] or zones['removed']):
+                if len(zones['added']) > 0:
+                    rgw_sync_policy_cmd += ['--zones', ','.join(zones['added'])]
 
-        if bucket_name:
-            rgw_sync_policy_cmd += ['--bucket', bucket_name]
+                    logger.info("Creating symmetrical flow! %s", rgw_sync_policy_cmd)
+                    try:
+                        exit_code, _, err = mgr.send_rgwadmin_command(rgw_sync_policy_cmd)
+                        if exit_code > 0:
+                            raise DashboardException(f'Unable to create sync flow: {err}',
+                                                     http_status_code=500, component='rgw')
+                    except SubprocessError as error:
+                        raise DashboardException(error, http_status_code=500, component='rgw')
 
-        try:
-            exit_code, _, err = mgr.send_rgwadmin_command(rgw_sync_policy_cmd)
-            if exit_code > 0:
-                raise DashboardException(f'Unable to create sync flow: {err}',
-                                         http_status_code=500, component='rgw')
-        except SubprocessError as error:
-            raise DashboardException(error, http_status_code=500, component='rgw')
+                if len(zones['removed']) > 0:
+                    rgw_sync_policy_cmd = ['sync', 'group', 'flow', 'remove', '--group-id',
+                                           group_id, '--flow-id', flow_id, '--flow-type',
+                                           SyncFlowTypes[flow_type].value]
+
+                    if bucket_name:
+                        rgw_sync_policy_cmd += ['--bucket', bucket_name]
+
+                    rgw_sync_policy_cmd += ['--zones', ','.join(zones['removed'])]
+
+                    logger.info("Removing symmetrical flow zones! %s", rgw_sync_policy_cmd)
+                    try:
+                        exit_code, _, err = mgr.send_rgwadmin_command(rgw_sync_policy_cmd)
+                        if exit_code > 0:
+                            raise DashboardException(f'Unable to remove sync flow: {err}',
+                                                     http_status_code=500, component='rgw')
+                    except SubprocessError as error:
+                        raise DashboardException(error, http_status_code=500, component='rgw')
```

Comment:
Instead of having to do all this on the `create_sync_flow` function, the removed code could lay over at the `remove_sync_flow` function which is doing kinda the same thing already. 
This would help with code duplication and readability.